GDJS/GDJS/Extensions/Builtin/Capacities/AnimatableExtension.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * GDevelop JS Platform
+ * Copyright 2008-2023 Florian Rival (Florian.Rival@gmail.com). All rights
+ * reserved. This project is released under the MIT License.
+ */
+#include "GDJS/Extensions/Builtin/Capacities/AnimatableExtension.h"
+#include "GDCore/Extensions/Builtin/AllBuiltinExtensions.h"
+#include "GDCore/Extensions/Metadata/InstructionMetadata.h"
+#include "GDCore/Tools/Localization.h"
+
+namespace gdjs {
+
+AnimatableExtension::AnimatableExtension() {
+  gd::BuiltinExtensionsImplementer::ImplementsAnimatableExtension(*this);
+
+  GetBehaviorMetadata("AnimatableCapability::AnimatableBehavior")
+      .SetIncludeFile("object-capabilities/AnimatableBehavior.js");
+
+  auto& actions = GetAllActionsForBehavior("AnimatableCapability::AnimatableBehavior");
+  auto& conditions = GetAllConditionsForBehavior("AnimatableCapability::AnimatableBehavior");
+  auto& expressions = GetAllExpressionsForBehavior("AnimatableCapability::AnimatableBehavior");
+  auto& strExpressions = GetAllStrExpressionsForBehavior("AnimatableCapability::AnimatableBehavior");
+
+  actions["AnimatableCapability::AnimatableBehavior::SetIndex"]
+      .SetFunctionName("setAnimationIndex")
+      .SetGetter("getAnimationIndex")
+      .SetIncludeFile("object-capabilities/AnimatableBehavior.js");
+  conditions["AnimatableCapability::AnimatableBehavior::Index"]
+      .SetFunctionName("getAnimationIndex")
+      .SetIncludeFile("object-capabilities/AnimatableBehavior.js");
+  expressions["Index"]
```

Comment:
Isn't the prefix `AnimatableCapability::AnimatableBehavior::` missing?

Reply:
(I'm saying this since I saw there is a prefix in the OpacityCapability expression)

Reply:
I'll remove it for the opacity. There is no prefix on expressions because the prefix is included in the grammar.